src/main/java/com/eleks/auctioneer/entity/AppUser.java

```diff
@@ -0,0 +1,73 @@
+package com.eleks.auctioneer.entity;
+
+import com.eleks.auctioneer.DTO.AppUserDTO;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "users")
+public class AppUser implements UserDetails {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id_user;
```

Comment:
You don't need to use underscore just because it's in the column name. Use naming strategy: https://www.baeldung.com/hibernate-field-naming-spring-boot or just @Column(name = ...)